File: Lat2/PHP/Product.php
<?php
class Product{
    private $price;
    private $id;
    function __construct(){
    }
    function setPrice($price){
        $this->price = $price;
    }
    function getPrice(){
        return $this->price;
    }
    function setId($id){
        $this->id = $id;
    }
    function getId(){
        return $this->id;
    }
    function __destruct()
    {
    }
}
?>

File: Lat2/C++/Product.cpp
#include <iostream>
#include <string>
using namespace std;
class Product{
    private:
        string price;
        string id;
    public:
        Product(){}
        void setPrice(string price) {
            this->price = price;
        }
        string getPrice() {
        return this->price;
        } 
        void setId(string id) {
            this->id = id;
        }
        string getId(){
            return this->id;
        }
        ~Product(){}
};